SteamKit2/SteamKit2/Steam/Discovery/SmartCMServerList.cs

```diff
@@ -53,6 +54,22 @@ public SmartCMServerList( SteamConfiguration configuration )
             this.configuration = configuration ?? throw new ArgumentNullException( nameof( configuration ) );
         }
 
+        /// <summary>
+        /// The default fallback Websockets server to attempt connecting to if fetching server list through other means fails.
+        /// </summary>
+        /// <remarks>
+        /// If the default server set here no longer works, please create a pull request to update it.
+        /// </remarks>
+        public static string DefaultServerWebsocket { get; set; } = "cmp1-ord1.steamserver.net";
```

Comment:
Can this config be moved to `SteamConfiguration` or does it need to be accessed w/o any configuration?

Reply:
server list has access to the config. I can move it but I will keep it static, the only reason I'm exposing it is for specific cases when someone wants to test it without rebuilding SK or something.